### PR TITLE
feat: landing page

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
image
image
image

### CODE REVIEW

File: src/pages/home/Home.jsx [hunk 1]
```diff
@@ -1,12 +1,59 @@
...
+          {featuredCategories.map(
+            ({ categoryName, _id, media, description }) => (
+              <div
+                className="card-container card-vertical grid-child"
+                title={categoryName}
+                key={_id}
+                onClick={() => featuredCategoryHandler(_id)}
+              >
+                <section className="card-media">
+                  <img src={media} alt="" />
```
Comment (rahulyadav139) [inline src/pages/home/Home.jsx]: Suggestion: alt tag should not be empty.

File: src/pages/home/Home.jsx [hunk 2]
```diff
@@ -1,12 +1,59 @@
...
+      <section>
+        <ImageSlider />
+        <h3>Featured Categories</h3>
+        <div className="grid-container grid-fit">
+          {featuredCategories.map(
+            ({ categoryName, _id, media, description }) => (
+              <div
+                className="card-container card-vertical grid-child"
+                title={categoryName}
+                key={_id}
```
Comment (rahulyadav139) [inline src/pages/home/Home.jsx]: Appreciation: you have destructured the map element before passing it to JSX. Great work.

**Review by rahulyadav139 — APPROVED** [review]
Codes are looking great. Readable. Great work.

File: src/pages/home/Home.jsx [hunk 3]
```diff
@@ -1,12 +1,59 @@
 import axios from "axios";
 import React from "react";
 import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useCategories } from "../../context/categories-context";
+import {ImageSlider} from "../../components/ImageSlider/ImageSlider";
```
Comment (ivinayakg) [inline src/pages/home/Home.jsx:6]: put the image in public folder then access it as a URL , read on accessing files in public folder in react

**Review by ivinayakg — APPROVED** [review]
great work, clean code 🙌

File: src/pages/home/Home.jsx [hunk 4]
```diff
@@ -1,12 +1,59 @@
...
-    </div>
+    <>
+      <h2>Meltin' Moments</h2>
+      <section>
+        <ImageSlider />
+        <h3>Featured Categories</h3>
+        <div className="grid-container grid-fit">
+          {featuredCategories.map(
+            ({ categoryName, _id, media, description }) => (
+              <div
```
Comment (ivinayakg) [inline src/pages/home/Home.jsx]: you can probably can make this into a component

**Comment by HarshnaNHaswani** [comment]
Thank you for your suggestions and review @ivinayakg  @rahulyadav139 , I'll make the required changes

**Pull request merged**